spec/ref_arch_setup/install_spec.rb

```diff
@@ -45,14 +47,14 @@
         it "returns true and outputs information about the bolt run" do
           expected_output = "All Good"
           expected_status = 0
-          expect(install).to receive(:`).with(@expected_command).and_return(expected_output)
+          expect(RefArchSetup::BoltHelper).to receive(:run_task_with_bolt)
+            .with("ref_arch_setup::install_pe", @env_vars, target_master).and_call_original
```

Comment:
Why call the original?

Reply:
(and if we change it not to, the context should change to match.)

Reply:
usually call_original is so other tests can depend upon its full functionality. 
or in the case of ruby functions (like `puts`) so rspec/pry itself can use it